sql/admin.md

```diff
@@ -189,3 +190,7 @@ ADMIN CANCEL DDL JOBS job_id [, job_id] ...
 * `ADMIN CANCEL DDL JOBS job_id [, job_id] ...`
 
     用于取消正在执行的 DDL 作业，其返回值为对应的作业取消是否成功，如果失败会显示失败的具体原因。这个操作可以同时取消多个 DDL 作业，其中 DDL 作业 ID 可以通过 `ADMIN SHOW DDL JOBS` 语句来获取。其中如果希望取消的作业已经完成，则取消操作将会失败。
+
+* `ADMIN CHECK TABLE tbl_name [, tbl_name] ...`
+
+	用于对给定表中的所有数据和对应索引进行一致性校验，若通过校验，则返回空的查询结果；否则返回 `data isn't equal` 错误。
```

Comment:
Please also update this line. Indent it with 4 spaces, not a Tab space. Otherwise, the CI will fail.

```suggestion
    用于对给定表中的所有数据和对应索引进行一致性校验，若通过校验，则返回空的查询结果；否则返回 `data isn't equal` 错误。
```